theme/tailwind.config.yml

```diff
@@ -12,3 +12,4 @@ colors:
     sub_navbar_text: '#ffffff'
     filters_navbar_bg: '#3D3D3D'
     filters_navbar_text: '#ffffff'
+  'cat-foobar': '#0800ff'
```

Comment:
This is where `cat-foobar` should live. But, atm, I need to also declare it in the global tailwind file for it to be picked up 🤷